src/pages/createOrder.js

```diff
@@ -359,6 +371,7 @@ class CreateOrderPage extends React.Component {
       collective: pick(this.props.data.Collective, ['id']),
       tier: tier ? pick(tier, ['id', 'amount']) : undefined,
       description: decodeURIComponent(this.props.description || ''),
+      customFields: customFields,
```

Comment:
Let's make sure we clarify `customFields` from `customValues`.

`customFields` is the schema coming out of the `Tier`.

`customValues` or `customData` (simply `key: value` formatted) is what is saved in the `Order`.

Here we want `customValues` or `customData`.